packages/cli/src/config/schema.ts

```diff
@@ -325,6 +325,16 @@ export const schema = {
 			default: 3600,
 			env: 'EXECUTIONS_DATA_PRUNE_TIMEOUT',
 		},
+
+		// Additional pruning option to delete executions if total count exceeds the configured max.
+		// Deletes the oldest entries first
+		// Default is 0 = No limit
+		pruneDataCountMax: {
```

Comment:
Nit: there's already `pruneDataMaxAge`, so for consistency between these two this ought to be `pruneDataMaxCount`

Reply:
updated